code/game/gamemodes/game_mode_latespawn.dm

```diff
@@ -44,3 +50,34 @@
 
 	message_admins("[uppertext(name)]: Failed to proc a viable spawn template.")
 	next_spawn = world.time + min_autotraitor_delay //recheck again in the miniumum time
+
+/datum/game_mode/proc/process_emergency_spawn()
+	message_admins("[uppertext(name)]: Checking to see if we need to add more antagonists...")
+	var/list/usable_templates = list()
+	for(var/datum/antagonist/A in antag_templates)
+		if(A.can_late_spawn() && A.can_emergency_spawn())
```

Comment:
Hmmmmmmm. It might be better to consider making `can_emergency_spawn()` return `can_late_spawn() && ...`. This would make the code a bit easier to modify per antag. And would compartmentalize the code while maintaining a sane standard.

Reply:
Though I do also acknowledge that, since we use the `late_spawn()` proc, the proc must explicitly meet the requirements for it (`can_late_spawn() == TRUE`). Which leads me to the next question, why are we restricting `ANTAG_OVERRIDE_MOB|ANTAG_OVERRIDE_JOB` flags from emergency spawning?

Reply:
ANTAG_OVERRIDE_MOB means that the antag is a mob that isn't human (malf, xeno, ect)
ANTAG_OVERRIDE_JOB means that the antag's job preferences must be overwritten before spawning them (merc, ninja, wizard, ect)

The system only works with on-station antags like traitor, rev, cult, and ling where crewmembers can keep their species and their job. If we did not have those checks, then weird things would happen.